ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -587,6 +587,28 @@ tests:
       ASSISTED_CONFIG: |
         IPXE_BOOT=true
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-reclaim-single-node
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    env:
+      ASSISTED_CONFIG: |
+        RECLAIM_HOSTS=true
+        ENABLE_KUBE_API=true
+        TEST_FUNC=test_late_binding_kube_api_sno
+        MAKEFILE_TARGET=test
```

Comment:
Maybe we'll have to change either the ``MAKEFILE_TARGET`` or the ``TEST`` value, because currently it's trying to discover the ``src/tests/test_e2e_install.py`` module and it's failing to make connection with the REST API (see [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/32302/rehearse-32302-pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-reclaim/1570219349308870656)):
```
waiting.exceptions.TimeoutExpired: Timeout of 900 seconds expired waiting for Wait till inventory is ready
```

So adding ``TEST=./src/tests/test_kube_api.py`` should fix it (also ``MAKEFILE_TARGET=test_kube_api_parallel`` like in the jobs above, but there are a couple of downsides in running single tests in a parallel-way)

Reply:
Oh right, thanks for catching this! Modified it to include `TEST=./src/tests/test_kube_api.py` Hopefully this will do it!